scripts/scr_unit_size/scr_unit_size.gml

```diff
@@ -1,23 +1,28 @@
 
-function scr_unit_size(armour, role, other_factors) {
+function scr_unit_size(armour, role, other_factors, mobility) {
 
     // armour: armour
     // role: role
     // other_factors: +?

```

Comment:
can remove above comments, the arguments are self-explanatory

Reply:
removed comments